scheduler/src/cook/config.clj

```diff
@@ -597,3 +597,20 @@
 (defn task-constraints
   []
   (get-in config [:settings :task-constraints]))
+
+(defn guard-invalid-gpu-models
```

Comment:
This is more a "guard-invalid-gpu-config" since it not only checks if the models are valid

Reply:
suggestion taken, thanks